src/App.jsx

```diff
@@ -14,70 +16,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam is selected');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam is selected');
```
It's a convention to name your state this way:
`[state, setState]`
`[whateverValueYouMayHave, setWhateverValueYouMayHave]`